zerver/views/message_edit.py

```diff
@@ -91,6 +92,35 @@ def get_message_edit_history(
 PROPAGATE_MODE_VALUES = ["change_later", "change_one", "change_all"]
 
 
+def validate_message_edit_payload(
+    message: Message,
+    stream_id: Optional[int],
+    topic_name: Optional[str],
+    propagate_mode: Optional[str],
+    content: Optional[str],
+) -> None:
+    if topic_name is None and content is None and stream_id is None:
```

Comment:
This probably deserves a comment that it's checking that it's a well-formed request, and does not consider questions of whether the user has permissions, which are done in `check_update_message`.

Reply:
Done!